src/convenience/frameworks.ts

```diff
@@ -26,7 +26,7 @@ export type SupportedFrameworks = keyof typeof adapters;
  * Abstraction over a request-response cycle, providing access to the update, as
  * well as a mechanism for responding to the request and to end it.
  */
-export interface ReqResHandler {
+export interface ReqResHandler<T = undefined> {
```

Comment:
I'm not sure if
```suggestion
export interface ReqResHandler<T = void> {
```
is possible, could you try?

Reply:
Seems to work.